custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -337,6 +344,92 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'case_xml', 'XML', is_default=True)
```

Comment:
I have updated the code to not rely on the format but on the url mentioned for the repeater, checking if its a WSDL link or not and presence of an operator.

So can just update this back to case_json and JSON but i feel its still confusing to keep both JSON/CASE JSON and SOAP together. Still feel keeping it XML makes more sense. let me know if its still confusing.